### PR TITLE
Improve API docs

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/2582

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Module.md [hunk 1]
```diff
@@ -324,12 +327,13 @@ sql:ParameterizedQuery query = `SELECT * FROM students
 stream<Student, sql:Error?> resultStream = dbClient->query(query);
 
 // Iterating the returned table.
-error? e = resultStream.forEach(function(Student student) {
-   //Can perform operations using the record 'student' of type `Student`.
-});
+check from Student student in resultStream
+    do {
+       // Can perform operations using the record 'student' of type `Student`.
+    }
```
Comment (niveathika) [inline ballerina/Module.md:333]:
```suggestion
    };
```

File: ballerina/Module.md [hunk 2]
```diff
@@ -344,10 +348,11 @@ sql:ParameterizedQuery query = `SELECT * FROM students
...
 // Iterating the returned table.
-error? e = resultStream.forEach(function(record{} student) {
-    // Can perform operations using the record 'student'.
-    io:println("Student name: ", student.value["name"]);
-});
+check from record{} student in resultStream 
+    do {
+        // Can perform operations using the record 'student'.
+        io:println("Student name: ", student.value["name"]);
+    }
```
Comment (niveathika) [inline ballerina/Module.md:355]:
```suggestion
    };
```

File: ballerina/Module.md [hunk 3]
```diff
@@ -426,10 +431,10 @@ sql:ProcedureCallResult result =
...
 stream<record{}, sql:Error?>? resultStr = result.queryResult;
 if resultStr is stream<record{}, sql:Error?> {
-    sql:Error? e = resultStr.forEach(function(record{} result) {
-      // Can perform operations using the record 'result'.
-    });
-}
+    check from record{} value in resultStr
+        do {
+          // Can perform operations using the record 'value'.
+        }}
```
Comment (niveathika) [inline ballerina/Module.md:437]:
```suggestion
        };
}
```

File: ballerina/Module.md [hunk 4]
```diff
@@ -426,10 +431,10 @@ sql:ProcedureCallResult result =
...
 stream<record{}, sql:Error?>? resultStr = result.queryResult;
 if resultStr is stream<record{}, sql:Error?> {
-    sql:Error? e = resultStr.forEach(function(record{} result) {
-      // Can perform operations using the record 'result'.
-    });
-}
+    check from record{} value in resultStr
+        do {
+          // Can perform operations using the record 'value'.
+        };
```
Comment (niveathika) [inline ballerina/Module.md:437]: Shouldnt the if block close here?

Comment (kaneeldias) [reply ballerina/Module.md:437]: Fixed in 85d98e3

File: ballerina/Module.md (hunk 3 above)
Comment (kaneeldias) [reply ballerina/Module.md]: Fixed in 85d98e3

File: ballerina/Module.md (hunk 2 above)
Comment (kaneeldias) [reply ballerina/Module.md]: Fixed in 85d98e3

File: ballerina/Module.md (hunk 1 above)
Comment (kaneeldias) [reply ballerina/Module.md]: Fixed in 85d98e3

**Review by niveathika — APPROVED** [review]

File: ballerina/Module.md [hunk 5]
```diff
@@ -71,9 +71,9 @@ mysql:Client|sql:Error dbClient = new (user = "rootUser", password = "rootPass",
 ```
 
 #### Using SSL
-To connect the MySQL database using an SSL connection, you must add the SSL configurations to the `mysql:Options` when creating the `dbClient`.
+To connect to the MySQL database using an SSL connection, you must add the SSL configurations to `mysql:Options` when creating the `mysql:Client`.
 For the SSL Mode, you can select one of the modes: `mysql:SSL_PREFERRED`, `mysql:SSL_REQUIRED`, `mysql:SSL_VERIFY_CA`, or `mysql:SSL_VERIFY_IDENTITY` according to the requirement.
-For the key and cert files, you must provide the files in the `.p12` format.
+For the key and cert files must be provided in the `.p12` format.
```
Comment (BuddhiWathsala) [inline ballerina/Module.md:76]:
```suggestion
The key and cert files must be provided in the `.p12` format.
```

Comment (kaneeldias) [reply ballerina/Module.md]: Fixed d31350b

**Review by BuddhiWathsala — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Pull request merged**